hurumap/urls.py

```diff
@@ -5,4 +5,4 @@
 from hurumap import settings
 
 urlpatterns = static(settings.STATIC_URL, document_root=settings.STATIC_ROOT) + \
-              wazimap_urlpatterns
+              list(wazimap_urlpatterns)
```

Comment:
If you haven't, add a story in PT to track this.

Reply:
@kilemensi I added a story for that